### DISCUSSION TITLE
X_tolerance in extract_table()

### DISCUSSION THREAD

**Discussion by manish291740** [jsvine/pdfplumber#653] [Ask for help with specific PDFs]
Hi Jsvine,

I want to extract data from table but in extraction I want to define X_tolerance in  extract_table() method. how can I achive this?

**Comment by jsvine** [comment]
Hi @manish291740, you can pass `text_x_tolerance` to the `.extract_table(...)` method: https://github.com/jsvine/pdfplumber#table-extraction-settings

<img width="285" src="https://user-images.githubusercontent.com/534702/167632395-4a45640e-a20f-4afc-a98a-b9eebd09ad16.png">
